lib/services/ios-project-service.ts

```diff
@@ -169,6 +171,89 @@ class IOSProjectService implements  IPlatformProjectService {
 		return this.$fs.exists(path.join(projectRoot, this.$projectData.projectName, constants.APP_FOLDER_NAME));
 	}
 
+    public addLibrary(platformData: IPlatformData, libraryPath: string): IFuture<void> {
+        return (() => {
+            this.validateDynamicFramework(libraryPath).wait();
+            var umbrellaHeader = this.getUmbrellaHeaderFromDynamicFramework(libraryPath).wait();
+
+            var frameworkName = path.basename(libraryPath, path.extname(libraryPath));
+            var targetPath = path.join(this.$projectData.projectDir, "lib", platformData.normalizedPlatformName, frameworkName);
+            this.$fs.ensureDirectoryExists(targetPath).wait();
+            shell.cp("-R", libraryPath, targetPath);
+
+            this.generateFrameworkMetadata(platformData.projectRoot, targetPath, frameworkName, umbrellaHeader).wait();
+
+            var pbxProjPath = path.join(platformData.projectRoot, this.$projectData.projectName + ".xcodeproj", "project.pbxproj");
+            var project = new xcode.project(pbxProjPath);
```

Comment:
It appears that we can use it without new: https://github.com/fealebenpae/node-xcode/blob/master/lib/pbxProject.js#L14